packtools/sps/validation/article_citations.py

```diff
@@ -29,23 +31,29 @@ def validate_article_citation_year(self):
                 },...
             ]
         """
+        if start_year is None:
+            start_year = 0
+        if end_year is None:
+            try:
+                end_year = int(ArticleDates(self.xmltree).collection_date['year'])
+            except TypeError:
+                raise ValidationArticleCitationsException('Article publication date not found and is required')
```

Comment:
@Rossi-Luciano terá que obter do `article_date['year']` porque de fato há casos em que não existe o valor para collection.

Reply:
OK, @robertatakenaka

Reply:
@Rossi-Luciano não. Estava correto antes, exceto que no lugar de `raise ValidationArticleCitationsException('Article publication date not found and is required')`, vc troca por ` end_year = int(ArticleDates(self.xmltree).article_date['year'])`